tst/EnergyPlus/unit/StandardRatings.unit.cc

```diff
@@ -432,6 +434,120 @@ TEST_F(EnergyPlusFixture, ChillerIPLVTestWaterCooled)
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff1 = 8.215998E-01;
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff2 = -2.209969E-02;
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff3 = -1.725652E-05;
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff4 = -3.831448E-02;
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff5 = 1.896948E-01;
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff6 = 2.308518E-02;
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff7 = 0;
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff8 = 1.349969E-02;
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff9 = 0;
+    state->dataCurveManager->PerfCurve(CurveNum).Coeff10 = 0;
+    state->dataCurveManager->PerfCurve(CurveNum).Var1Min = 17.52;
+    state->dataCurveManager->PerfCurve(CurveNum).Var1Max = 33.32;
+    state->dataCurveManager->PerfCurve(CurveNum).Var2Min = 0.10;
+    state->dataCurveManager->PerfCurve(CurveNum).Var2Max = 1.08;
+    state->dataChillerReformulatedEIR->ElecReformEIRChiller(1).ChillerEIRFPLRIndex = 3;
+
+    state->dataPlnt->TotNumLoops = 1;
+    state->dataPlnt->PlantLoop.allocate(state->dataPlnt->TotNumLoops);
+    state->dataPlnt->PlantLoop.allocate(state->dataPlnt->TotNumLoops);
+    auto &loopside(state->dataPlnt->PlantLoop(1).LoopSide(DataPlant::LoopSideLocation::Demand));
+    loopside.TotalBranches = 1;
+    loopside.Branch.allocate(1);
+    auto &loopsidebranch(state->dataPlnt->PlantLoop(1).LoopSide(DataPlant::LoopSideLocation::Demand).Branch(1));
+    loopsidebranch.TotalComponents = 1;
+    loopsidebranch.Comp.allocate(1);
+    state->dataPlnt->PlantLoop(1).Name = "ChilledWaterLoop";
+    state->dataPlnt->PlantLoop(1).FluidName = "ChilledWater";
+    state->dataPlnt->PlantLoop(1).FluidIndex = 1;
+    state->dataPlnt->PlantLoop(1).PlantSizNum = 1;
+    state->dataPlnt->PlantLoop(1).FluidName = "WATER";
+
+    Real64 IPLV;
+    CalcChillerIPLV(*state,
+                    state->dataChillerReformulatedEIR->ElecReformEIRChiller(1).Name,
+                    DataPlant::PlantEquipmentType::Chiller_ElectricReformEIR,
+                    state->dataChillerReformulatedEIR->ElecReformEIRChiller(1).RefCap,
+                    state->dataChillerReformulatedEIR->ElecReformEIRChiller(1).RefCOP,
+                    state->dataChillerReformulatedEIR->ElecReformEIRChiller(1).CondenserType,
+                    state->dataChillerReformulatedEIR->ElecReformEIRChiller(1).ChillerCapFTIndex,
+                    state->dataChillerReformulatedEIR->ElecReformEIRChiller(1).ChillerEIRFTIndex,
+                    state->dataChillerReformulatedEIR->ElecReformEIRChiller(1).ChillerEIRFPLRIndex,
+                    state->dataChillerReformulatedEIR->ElecReformEIRChiller(1).MinUnloadRat,
+                    IPLV,
+                    state->dataChillerReformulatedEIR->ElecReformEIRChiller(1).CondVolFlowRate,
+                    1,
+                    1.0);
+
+    EXPECT_DOUBLE_EQ(round(IPLV * 100) / 100, 4.83);
+}
+
```

Comment:
New unit test for the IPLV calculation of reformulated EIR chillers.

Reply:
I :heart: unit tests that just build up the required components rather than require the input blob.  Very nice.